utilities/after_success.sh

```diff
@@ -31,6 +31,8 @@ if [ "${TRAVIS_JDK_VERSION}" == "oraclejdk7" -a "${TRAVIS_BRANCH}" == "master" -
         # Update versions README and pom.xml in master branch
```

Comment:
I thing here we are only updating the READMEs, can you confirm that? If so would you mind updating the comment?